backend/webserver/models.py

```diff
@@ -15,3 +16,61 @@ class FollowRequest(models.Model):
 class Follow(models.Model):
     follower = models.ForeignKey(Author, related_name='following_authors', on_delete=models.CASCADE)
     followee = models.ForeignKey(Author, related_name='followed_by_authors', on_delete=models.CASCADE)
+
+class Post(models.Model):
+
+    author = models.ForeignKey(Author,on_delete=models.CASCADE)
+
+    created_at = models.DateTimeField(verbose_name="date created",null=False, blank=False, default = datetime.now)
+
+    edited_at = models.DateTimeField("date edited")
+
+    title = models.CharField(max_length=200)
+
+    description = models.CharField(max_length=200)
```

Comment:
```suggestion
    description = models.TextField(blank=True)
```
- [We should allow longer descriptions](https://docs.djangoproject.com/en/4.1/ref/models/fields/#textfield)
- `blank=True` because frontend could send us an empty string if post has no description. Our job is to remain flexible.